dockers/docker-sonic-mgmt-framework/Dockerfile.j2

```diff
@@ -7,15 +7,23 @@ RUN [ -f /etc/rsyslog.conf ] && sed -ri "s/%syslogtag%/$docker_container_name#%s
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y g++ python-dev libxml2
+    apt-get install -y g++ python-dev python3-dev libxml2
```

Comment:
You can probably also remove `python3-dev` on line 43, as it is most likely not needed after we install the Python packages. Just help keep the container size down a bit.

Reply:
@jleveque please review again. I have addressed the review comments

Reply:
So you don't need to install the package at all? I was suggesting to remove the package during cleanup on line 43, but if you have proven that it doesn't need to be installed at all, this is better.

Reply:
Discussed with other contributors, it seems it's needed. Added it back. Addressed pip to pip2 comments. please re-review. thanks, in-advance